language-server/src/util/util.js

```diff
@@ -108,3 +113,40 @@ export const readDirRecursive = async function* (path, filter, cwd) {
     }
   }
 };
+
+/** @type (textDocument: TextDocument, textEdit: TextEdit, settings: IndentationSettings) => TextEdit */
```

Comment:
```suggestion
/** @type (textDocument: TextDocument, textEdit: TextEdit, settings: DocumentSettings) => TextEdit */
```

I'm surprised we're not getting a type error here. What you're passing is the result of calling `Configuration.get`, which is `DocumentSettings`. Since you didn't update `DocumentSettings` with the all the indentation settings, I would have expected it to tell you that the value passed is not compatible with `IndentationSettings`. I think it slips through the cracks just because none of the `IndentationSettings` are required.